### DISCUSSION TITLE
Local API Error

### DISCUSSION THREAD

**Discussion by HiImTemmy** [FreeTubeApp/FreeTube#4703] [General]
Hello,

Using FreeTube (v0.19.1 Beta) these last few days is throwing an error when searching. 

![test](https://github.com/FreeTubeApp/FreeTube/assets/83087378/0f26921c-c218-43c8-8b21-5b046d73181b)

**Comment by Lence23** [comment]
I solved this issue by.  Settings > General Settings > chose Invidious API from 'Preferred API backend'  and 'click'  "Current Instance as Default"  button.

if it slow just chose another 'invidious instance'.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Duplicate of #4700

**Reply by Lence23** [reply]
I don't know if it an issue here, I am new to github.
